SimPEG/Utils/PlotUtils.py

```diff
@@ -99,13 +108,8 @@ def plot2Ddata(
             MASK = MASK.reshape(X.shape)
             DATA = np.ma.masked_array(DATA, mask=MASK)
 
-        if 'vmin' not in contourOpts.keys():
-            contourOpts['vmin'] = vmin
-        if 'vmax' not in contourOpts.keys():
-            contourOpts['vmax'] = vmax
-
         cont = ax.contourf(
-            X, Y, DATA, levels=levels,
+            X, Y, DATA, levels=levels, vmin=vmin, vmax=vmax,
```

Comment:
@lheagy : Will this work? If vmin, vmax were in ContourOpts, you will get an error because vmin and vmax will be specified twice. That was the reason for the section of code you deleted:
`if 'vmin' not in contourOpts.keys():
            contourOpts['vmin'] = vmin
        if 'vmax' not in contourOpts.keys():
            contourOpts['vmax'] = vmax`

Reply:
when you use `contourOpts.pop("vmin")` it will remove vmin from the `contourOpts` dictionary (see lines 48, 57). Does this make sense?

Reply:
![image](https://user-images.githubusercontent.com/6361812/50803864-5dd7c580-12a0-11e9-8878-f07eb2fd233d.png)

Reply:
Thanks @lheagy , yes it makes sense :)